fixtures/provider.py

```diff
@@ -292,6 +292,9 @@ def template(template_location, provider):
 
 def _get_template(provider, template_type_name):
     template = provider.data.get(template_type_name)
+    # if the template is None, try to get it from templates section as per new YAMLs
```

Comment:
This comment should not discuss the newness of the feature but the feature itself.   So the sentence fragment, "try to get it from the templates section" should continue on with some like "because ..."   and fill in the elipsis with why we get it from there (i.e. what is the purpose of this templates section).